server/routes/search/controller.ts

```diff
@@ -2,15 +2,85 @@ import { Request, Response } from 'express'
 import { Page } from '../../services/auditService'
 import { PageHandler } from '../../interfaces/pageHandler'
 import SEARCH_PRISONER_URL from '../urls'
+import PrisonerSearchService from '../../services/prisonerSearchService'
+import config from '../../config'
+import logger from '../../../logger'
+import { getResultsPagingLinks } from '../../utils/utils'
 
 export default class SearchController implements PageHandler {
+  constructor(private readonly prisonerSearchService: PrisonerSearchService) {}
+
   public PAGE_NAME = Page.SEARCH_PRISONERS_PAGE
 
   GET = async (req: Request, res: Response): Promise<void> => {
-    res.render('search/view', { validationErrors: res.locals.validationErrors })
+    let prisoners = null
+    let pageLinks: object
+    let to: number
+    let parsedPage: number
+    let prisonerNotFoundMessage: string
+    if (req.query.search) {
+      req.session.search = null
+    }
+    const search = req.session.search ? req.session.search : req.query.search
+    const { pageSize } = config.apis.prisonerSearch
+    try {
+      if (res.locals.validationErrors === undefined && search) {
+        const currentPage = typeof req.query.page === 'string' ? req.query.page : ''
+        parsedPage = Number.parseInt(currentPage, 10) || 1
+
+        // TODO: This value should really be set against the res.locals.user ojbect, as `activeCaseloadId`
+        // populateCurrentUser middleware is the place to set these things.
+        req.session.prisonId = 'HEI'
+
```

Comment:
This will be addressed in a follow-on PR - to set from the users currently activeCaseloadId, whether that is in the Session Data or in the User object.